### ISSUE TITLE
programs are not started into background with Type=simple

### ISSUE THREAD

**Issue by stapelberg** [akhilvij/systemd-to-sysvinit-converter#7]
I’m using the following service file:

```
[Unit]
Description=i3 IRC bot to announce buildbot status
After=network.target

[Service]
ExecStart=/usr/bin/go-buildbot-announce
User=build

[Install]
WantedBy=multi-user.target
```

As you can see, there is no `Type=` specified in `[Service]`. According to `systemd.service(5)`, this means that the Type is `simple`, meaning that the program does not fork into background itself.

When I convert the service file to an initscript, the `start_daemon` call does not account for that: It is using neither `--background` nor `--make-pidfile` when calling `start-stop-daemon`. Therefore, when starting the generated initscript, my terminal controls the process and pressing Ctrl-C aborts it.

**Comment by akhilvij** [comment]
Yeah, I had a discussion with the mentors regarding the same issue. The solutions proposed were:
- Calling start-stop-daemon instead of relying on lsb init-functions.
- Use your own copy of customized init-functions.
- Request upstream for including an option to allow arguments for start-stop-daemon through start_daemon.

According to me, second one looks like a better option. Any thoughts?

**Comment by stapelberg** [comment]
I’d prefer option 1 since it’s more consistent with the init scripts that we currently have in Debian.
